src/conf_mode/container.py

```diff
@@ -0,0 +1,177 @@
+
+    if 'name' in container or 'network' in container:
+        container['configured'] = True
+
+    # Delete container network, delete containers
+    dict = {}
+    tmp_net = node_changed(conf, ['container', 'network'])
+    if tmp_net:
+        dict = dict_merge({'net_remove' : tmp_net}, dict)
+        container.update(dict)
+
+    tmp_name = node_changed(conf, ['container', 'name'])
+    if tmp_name:
+        dict = dict_merge({'container_remove' : tmp_name}, dict)
+        container.update(dict)
+
+    return container
+
+def verify(container):
+    # bail out early - looks like removal from running config
+    if not container:
+        return None
+    else:
+        # Start docker
+        docker_pid = process_named_running('dockerd')
+        if not docker_pid:
+            call('systemctl restart docker.service')
+
+    # Add new container
+    if 'name' in container:
+        for name in container['name']:
+            # Dont add container with wrong/undefined name network
+            if 'network' in container['name'][name] and 'network' in container:
+                if list(container['name'][name]['network'])[0] not in container['network']:
+                    # Need to refrase. Don't allow add container with wrong network name.
+                    # Don't allow delete network if container use that network.
+                    raise ConfigError('Netowrk with name: {0} shuld be specified!'.format(list(container['name'][name]['network'])[0]))
+
+            # If image not defined
+            if 'image' not in container['name'][name]:
+                raise ConfigError(f'Image for \'{name}\' is required!')
+
+            # Image should be downloaded before we run the container. Or not?.
+            # For example 'docker image ls --format {{.Repository}} --filter "reference=alpine"'
+            if 'image' in container['name'][name]:
+                image = container['name'][name]['image']
+                repo = cmd(f'docker image ls --format ' + '{{.Repository}}' + f' --filter \"reference={image}\"')
+                # Check if image was pulled and present in local images
+                if image != repo:
+                    raise ConfigError(f'Pull image for container \'{name}\' is required!\nUse: run add container image {image}')
+
+            # If 'allow-host-networks' or 'network' not set.
+            if 'allow_host_networks' not in container['name'][name] and 'network' not in container['name'][name]:
+                raise ConfigError(f'\n\'Network\' or \'allow-host-networks\' for \'{name}\' is required!')
+
+            # If set both parameters for networks (host and user-defined). We require only one.
+            if 'allow_host_networks' in container['name'][name] and 'network' in container['name'][name]:
+                raise ConfigError(f'\n\'allow-host-networks\' and \'networks\' for \'{name}\' cannot be both configured at the same time!')
```

Comment:
`raise ConfigError(f'"allow-host-networks" and "networks" for "{name}" cannot be both configured at the same time!')`

We ususally use double quotes

Reply:
done

> Please also add your Python handler to `configd-include.json` so it can be run under vyos-configd.

done